public/maintenance.html

```diff
@@ -87,8 +87,8 @@
   <body>
     <div class="dialog">
       <div>
-        <h1>Site is currently down for maintenance.</h1>
-        <p>We are making some changes and will be back shortly, so please try again soon.</p>
+        <h1>Site is currently down for <%= reason ? reason : "maintenance" %> as of <%= Time.now.strftime("%H:%M %Z") %>.</h1>
```

Comment:
I'd prefer and have used this in a few projects now:

```html
<h1>Site is currently in maintenance mode.</h3>
<p>Sorry for the inconvenience, but the system is down for <%= reason ? reason : "maintenance" %> as of <%= Time.now.strftime("%H:%M %Z") %>.</p>
<p> We are making some changes and will be back online <%= deadline ? deadline : "shortly" %>, so please try again soon!</p>
<p>Thanks for your patience.</p>

```

For some, I have also added a "contact us" mailto link. However, that can be added/changed later if required.